app/modules/listings/lib/search_types.ts

```diff
@@ -0,0 +1,27 @@
+import { typesBySection } from '#search/lib/search_sections'
```

Comment:
This file seems to be an obsolete duplicate of `app/modules/listings/lib/entities_typing.ts`